signature/policy_config_test.go

```diff
@@ -152,35 +152,41 @@ func TestDefaultPolicyPath(t *testing.T) {
 	const rootPrefix = "/root/prefix"
 	tempHome := t.TempDir()
 	userDefaultPolicyPath := filepath.Join(tempHome, userPolicyFile)
-
+	tempsystemdefaultpath := filepath.Join(tempHome, systemDefaultPolicyPath)
 	for _, c := range []struct {
-		sys             *types.SystemContext
-		userfilePresent bool
-		expected        string
+		sys                  *types.SystemContext
+		userfilePresent      bool
+		expected             string
+		createSystemTempFile bool
```

Comment:
Please move this one line above, so that the “test inputs” go before  “expected outputs” . Also, the `userfilePresent`/`createSystemTempFile` options should have a similarly structured name — I don’t care much _what_ the name is, but it should not suggest that the two are different kinds of things.